anaconda.py

```diff
@@ -426,8 +426,8 @@ def _earlyExceptionHandler(ty, value, traceback):
         wait_for_connected_NM(timeout=opts.waitfornet)
 
     # In any case do some actions only after NM finishes its connecting.
-    threadMgr.add(AnacondaThread(name=constants.THREAD_WAIT_FOR_CONNECTING_NM,
-                                 target=wait_for_connecting_NM_thread))
+    thread_manager.add(AnacondaThread(name=constants.THREAD_WAIT_FOR_CONNECTING_NM,
```

Comment:
If you want to improve the API, I would consider adding support for `thread_manager.add_thread()`.

Instead of this call:

```
thread_manager.add(AnacondaThread(
   name=constants.THREAD_WAIT_FOR_CONNECTING_NM,
   target=wait_for_connecting_NM_thread
))
```

You could call something like this:

```
thread_manager.add_thread(
   name=constants.THREAD_WAIT_FOR_CONNECTING_NM,
   target=wait_for_connecting_NM_thread
)
```